test/unit/test_dss_client.py

```diff
@@ -335,6 +337,20 @@ def _assert_bundle_json(self, prefix=''):
             actual_hash = hashlib.sha256(f.read()).hexdigest()
         self.assertEqual(actual_hash, expected_hash)
 
+    @unittest.skipIf(os.name is 'nt', 'os.link is not used on Windows')
+    def test_link_fails(self):
```

Comment:
Is it possible to refactor parts of `TestManifestDownloadBundle.test_link_fails()` and `TestDownload.test_link_fails()` into a fixture?

Reply:
Don't understand what you mean there.